CODEOWNERS

```diff
@@ -1,4 +1,52 @@
 # Users referenced in this file will automatically be requested as reviewers for PRs that modify the given paths.
 # See https://help.github.com/articles/about-code-owners/
 
-* @dotnet/dotnet-docker-reviewers
+### General infra ###
+/* @dotnet/dotnet-docker-reviewers

```

Comment:
I think the * reviewer should be retained. As it stands if someone modified a manifest but nothing else, no reviewers would be added.  The docker reviewers team is a good general reviewer for patterns/practices/infra.

Reply:
Sounds good - I'll add functionality to not include this pattern when validating code owners then. Otherwise, all of the Dockerfiles would have `@dotnet/dotnet-docker-reviewers` as a codeowner + all other validation would essentially be obsolete.